packages/SwingSet/src/kernel/kernel.js

```diff
@@ -755,9 +762,9 @@ export default function buildKernel(
       return { vatID, targetObject };
     }
 
-    function enqueue() {
-      kernelKeeper.addMessageToPromiseQueue(target, msg);
-      return null; // message is queued, not sent to a vat right now
+    function requeue() {
+      // message will be requeued, not sent to a vat right now
+      return { vatID: null, targetObject: target };
```

Comment:
If the `targetObject` return value can actually be a Promise (kpid), not just an Object (koid), we should rename it.

We might be able to just use `target` for the output, since in many cases it is unchanged. It depends upon the caller, but if they hold the result in `route` (instead of immediately unpacking it into `{ vatID, target } = routeSendEvent()`), then the difference between the input `target` and the output `route.target` is pretty clear.

If not, maybe `routeTarget` or `routedTarget`.